services/gateway/app.js

```diff
@@ -75,6 +75,10 @@ tempApp.run().then(async () => {
 	const registeredModuleNames = registeredModules.map(
 		module => module.name === MODULE.REWARD ? MODULE.DYNAMIC_REWARD : module.name,
 	);
+
+	// TODO: remove after getSystemMetadata returns NFT
+	registeredModuleNames.push('nft');
```

Comment:
same question for this one. can we add this logic to connector side so that we do not need to duplicate the logic?